Resources/Prototypes/Entities/Structures/Machines/Computers/base_structurecomputers.yml

```
@@ -5,6 +5,15 @@
   placement:
     mode: SnapgridCenter
   components:
+  # Fire added start
+  - type: AmbientOnPowered
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 4 but found 2

(indentation)